src/test/java/ticketbooking/TicketBookingApplicationTests.java

```diff
@@ -0,0 +1,12 @@
+package ticketbooking;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class TicketBookingApplicationTests {
+
+	@Test
+	void contextLoads() {
```

Comment:
i to jjuż? przetestowane?

Reply:
Przyznaję, poskąpiłem trochę na testach( 2 są :) ) - ale po ostatnim feedbacku miałem też trochę wątpliwości co konkretnie testować a co nie.